pig/src/main/kotlin/org/partiql/pig/domain/parser/ParserErrorContext.kt

```diff
@@ -77,6 +78,9 @@ sealed class ParserErrorContext(val msgFormatter: () -> String) : ErrorContext {
 
     object MultipleElementIdentifierAnnotations :
         ParserErrorContext({ "Element has multiple name annotations" })
+
+    object MultipleTypeAnnotations :
+        ParserErrorContext({ "Only one TypeAnnotation (${TypeAnnotation.values().joinToString()}) is supported" })
```

Comment:
A node only receives one annotation for now; since a node cannot be both experimental and deprecated. The link you referred to is marking the **experimental annotation** itself as deprecated.

Reply:
Sorry about the miss. Then yes make sense.